Makefile

```diff
@@ -57,7 +57,7 @@ devdata: python
 
 .PHONY: shell
 shell: python
-	@tox -qe dev -- sh bin/hypothesis --dev shell
+	@pyenv exec tox -qe dev --run-command 'pshell conf/development-app.ini'
```

Comment:
This makes the `shell` recipe in h's `Makefile` the same as the one in the cookiecutter's `Makefile` template, which is necessary in order to apply the cookiecutter to h.